core/src/test/java/feign/FeignUnderAsyncTest.java

```diff
@@ -490,7 +490,7 @@ void throwsFeignExceptionIncludingBody() {
     } catch (FeignException e) {
       assertThat(e.getMessage())
           .isEqualTo("timeout reading POST http://localhost:" + server.getPort() + "/");
-      assertThat(e.contentUTF8()).isEqualTo("Request body");
```

Comment:
So, backwards compatibility is my main concern here.

I think we need to deprecate `contentUTF8`, keep response body as result of `contentUTF8` and introduce `resquestBody` and `responseBody` or something with that meaning.

Breaking compatibility like this only on major releases.

Reply:
It looks like there is also the same bug, because `contentUTF8()` method contains following code:
```
public String contentUTF8() {
    if (responseBody != null) {
      return new String(responseBody, UTF_8);
    } else {
      return "";
    }
  }
```
Also `content()` method is deprecated and method `responseBody()` already exists.

Reply:
For example in this [case](https://github.com/OpenFeign/feign/blob/2396c4b51d9e57e097b300b007907394dcc18186/core/src/test/java/feign/codec/DefaultErrorDecoderTest.java#L61) `contentUTF8()` returns response body `hello world`